.github/workflows/spellcheck.yml

```
@@ -0,0 +1,16 @@
+name: Spellcheck
+on:
+  pull_request:
+    paths:
+      - '**/*.md'
+      - '**/*.go'
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)